### DISCUSSION TITLE
Tutorial using django generic class based views (CBV's)

### DISCUSSION THREAD

**Discussion by alialjowaher** [viewflow/viewflow#465] [Ideas]
Hello Everyone, I would like to see a tutorial showcasing how to integrate the library within my Django app using Django generic views if possible . I tried the cookbook but it's not working correctly when i try to build my custom UI ,  some help is much appreciated

**Comment by kmmbvnr** [comment]
Using class-based-views covered at https://docs.viewflow.io/workflow/custom_views.html#class-based-views

Any view from flow.View(...) got request.activation instance, and you need to call request.activation.execute() in your form_valid method

**Reply by alialjowaher** [reply]
that is  the issue `request.activation.execute()`  keeps on giving WSGI error , as if the method is not in the request i'm using django 4.2 and tried also n 5.0 . keeps on confusing me. I'll try to setup a new project later today and get back with an update, thanks for your time and assistance
